.github/workflows/licenseCheck.yml

```diff
@@ -0,0 +1,42 @@
+name: Check licenses
+
+on:
+  push:
+    branches:
+    - master
+    - beta
+    - rc
+    - try-**
+    - release-**
+    - githubActions # To be removed
+
+  pull_request:
+    branches:
+    - master
+    - beta
+    - rc
+    - try-**
+    - release-**
+
+  workflow_dispatch:
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+env:
+  if: $GITHUB_EVENT_NAME == 'pull_request'
+  pullRequestNumber: ${{ github.event.number }}
+jobs:
+  licenseCheck:
+    runs-on: windows-latest
+    defaults:
+      run:
+        shell: cmd
+    steps:
+    - name: Checkout NVDA
+      uses: actions/checkout@v4
+      with:
+        submodules: true
+    - name: Build NVDA
```

Comment:
I don't think we need a build here either. Just install licensecheck and run it on Linux.